src/scenes/intro-and-main-menu/intro.ts

```diff
@@ -1,89 +1,49 @@
   constructor() {
     super({ key: 'Intro' });
   }
 
   preload() {
   }
 
+  public init(data: any) {
+    super.init(data);
+  }
+
   create() {
     this.cameras.main.fadeIn(2000, 0, 0, 0);
 
     const soundManagerScene = this.scene.get('BackgroundSound') as BackgroundSoundScene;
     soundManagerScene.playBackgroundMusic('intro');
 
-    /*    const bgVoice = this.sound.add('intro-voice', {
-      volume: 0, // currently disabled since audio probably should be re-recorded
-    });
+    this.scene.run('StoryTellingWindow');
 
-    bgVoice.play(); */
-
-    this.add.text(GAME_W / 2, GAME_H / 2 - 70,
-      'Still... Stay still... Calm... Stay calm... \n\n'
-      + 'You do not have to go, you do not have to do anything at all...\n\n'
-      + 'Just stay that incredibly still..and listen to my voice.\n\n\n'
-      + 'I am here to help. I Can help you, and I Will. But I need you to help me first.\n\n'
-      + 'Concentrate.. Concentrate on my voice, stay still...and Think. \n\n'
-      + 'I need you to help me, I need you to give me a Reason.\n\n'
-      + 'A reason to help you.\n\n'
-      + 'So ... Think. Just Think...\n\n\n'
-      + 'Good... Very good.\n\n\n'
-      + 'Now...take me there, show me...',
-      {
-        font: '14px monospace',
-        color: '#ffffff',
-      })
-      .setOrigin(0.5, 0.5);
-
-    const showHimVillage = this.add.text(GAME_W / 2, GAME_H / 2 + 200,
-      'Show him that lovely morning at the Village',
-      {
-        font: '14px monospace',
-        color: '#ffffff',
-        backgroundColor: '#222222',
-        padding: {
-          x: 10,
-          y: 10,
+    this.time.delayedCall(7500, () => {
+      this.switchToScene('Dialog', {
+        dialogTree: introRevivalDialog,
+        speakerName: 'Unknown',
+        sounds: {
+          volume: 0.1,
+          soundKeys: { shortSound: 'typewriter-intro-short', longSound: 'typewriter-intro-long', endSound: 'typewriter-end' },
         },
-      })
-      .setOrigin(0.5, 0.5)
-      .setInteractive({ useHandCursor: true });
-
-    this.add.graphics()
-      .lineStyle(2, 0xffffff, 0.4)
-      .strokeRect(showHimVillage.getTopLeft().x, showHimVillage.getTopLeft().y, showHimVillage.width, showHimVillage.height);
-
-    showHimVillage.once('pointerdown', () => {
-      // bgVoice.stop();
-      this.scene.start('WeaklingsCave', { prevScene: this.scene.key });
-    });
-
-    const showHimDungeon = this.add.text(GAME_W / 2, GAME_H / 2 + 250,
-      'Show him the dungeon',
-      {
-        font: '14px monospace',
-        color: '#ffffff',
-        backgroundColor: '#222222',
-        padding: {
-          x: 10,
-          y: 10,
+        closeCallback: () => {
+          this.cameras.main.fadeOut(1500, 0, 0, 0);
+          this.scene.remove('StoryTellingWindow');
+          this.time.delayedCall(1500, () => {
+            this.scene.start('WeaklingsCave', { prevScene: this.scene.key });
+          });
```

Comment:
Camera has FADE_OUT_COMPLETE event on it, mb would be better to use that instead of timer? https://newdocs.phaser.io/docs/3.55.2/Phaser.Cameras.Scene2D.Events.FADE_OUT_COMPLETE